docs/content/manual/_incoming/start-vm-after-harvester-upgrade-complete.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Check can start VM after Harvester upgrade
+---
+
+* Related issues: 
```

Comment:
If there's no related issues you should probably remove this line

Reply:
Thanks for the check. There should be a related issue https://github.com/harvester/harvester/issues/2270 since I forget to add them on it.